src/guides/v2.3/migration/migration-migrate-settings.md

```diff
@@ -52,17 +52,17 @@ You may ignore, rename or change the system configurations when migrating settin
 
 1. Change to the following directory:
 
-    ```bash
-    cd <your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/<edition-to-edition>
-    ```
+   ```bash
+   cd <your Magento 2 install dir>/vendor/magento/data-migration-tool/etc/<edition-to-edition>
+   ```
 
-    For example, if Magento 2 is installed in `/var/www/html`, you will find `settings.xml.dist` in one of the following directories:
+   For example, if Magento 2 is installed in `/var/www/html`, you will find `settings.xml.dist` in one of the following directories:
 
-    `/var/www/html/vendor/magento/data-migration-tool/etc/opensource-to-commerce`
+   `/var/www/html/vendor/magento/data-migration-tool/etc/opensource-to-commerce`
```

Comment:
I'm not sure that it will be rendered as a list. This should probably be converted to an unordered list.

Reply:
done